dolphinscheduler-alert-plugin/dolphinscheduler-alert-dingtalk/src/main/java/org/apache/dolphinscheduler/plugin/alert/dingtalk/DingTalkAlertChannelFactory.java

```diff
@@ -45,11 +45,6 @@ public String getName() {
                 .setRequired(true)
                 .build())
             .build();
-        InputParam keywordParam = InputParam.newBuilder(DingTalkParamsConstants.NAME_DING_TALK_KEYWORD, DingTalkParamsConstants.DING_TALK_KEYWORD)
```

Comment:
If you delete this field, the message may not be sent out.

Reply:
Is testing，Wait me the result

Reply:
![image](https://user-images.githubusercontent.com/59079269/106275460-d6879100-6270-11eb-84ff-87cca5ccdcbf.png)
i see